pyiceberg/types.py

```diff
@@ -175,6 +179,10 @@ def is_primitive(self) -> bool:
     def is_struct(self) -> bool:
         return isinstance(self, StructType)
 
+    def minimum_format_version(self) -> int:
```

Comment:
Let's throw in a nice alias:
```suggestion
    def minimum_format_version(self) -> TableVersion:
```

Reply:
Sounds good! Thanks for the suggestion